src/log.ts

```diff
@@ -13,7 +13,7 @@ export const logPolicyResult = (result: Result) => {
   if (!result.policy.isCountAcceptable(result.matches)) {
     return console.error(
       `${RED_X} ${chalk.red.bold(result.policy.name)}: ${result.matches.length} (expected ${result.policy.baseline
-      } or fewer)`
+      }${result.policy.baselineStrictEqual ? "" : " or fewer"})`
```

Comment:
I'd put it as " exactly" : " or fewer" (just thinking of people coming to this without reading carefully).

Reply:
💡